2-mandatory/7-lane-names.js

```diff
@@ -6,7 +6,11 @@
   HINT: string and array methods that could be helpful (indexOf, filter)
 */
 
-function getLanes() {}
+
+function getLanes(names) {
+  let result = names.filter((array) => array.includes("Lane"));
+  return result;
+}
```

Comment:
Good use of .filter and .includes method here well done :) Let me know if you want to go through some of the other Journey Planner questions that you didn't get onto